scripts/new-migration.sh

```diff
@@ -2,9 +2,17 @@
 # To run it:
 # sh scripts/new-migration.sh
 
+container_name=sl-db-new-migration
+
+if [ "$#" -lt "1" ]; then
+	echo "What is this migration for?"
```

Comment:
Does this mean we require the developer to give a name for a migration? Usually I never modify a migration file and leave alembic handle everything.

Reply:
I'll make it optional.